pkg/lib/config/user_profile.go

```diff
@@ -224,6 +221,18 @@ func (c *StandardAttributesConfig) GetAccessControl() accesscontrol.T {
 	return t
 }
 
+func (c *StandardAttributesConfig) IsEndUserAllHidden() bool {
+	if len(c.AccessControl) <= 0 {
```

Comment:
After updating the `SetDefaults` implementation, the length of `AccessControl` will never be zero?